baby-liminal-extension/src/frontend.rs

```diff
@@ -71,20 +60,11 @@ impl ink::env::chain_extension::FromStatusCode for BabyLiminalError {
 pub trait BabyLiminalExtension {
     type ErrorCode = BabyLiminalError;
 
-    /// Directly call `pallet_baby_liminal::store_key`.
-    // IMPORTANT: this must match the extension ID in `extension_ids.rs`! However, because constants
-    // are not inlined before macro processing, we can't use an identifier from another module here.
-    #[ink(extension = 41)]
-    fn store_key(
-        origin: AccountId,
-        identifier: VerificationKeyIdentifier,
-        key: Vec<u8>,
-    ) -> Result<(), BabyLiminalError>;
-
-    /// Directly call `pallet_baby_liminal::verify`.
+    /// Verify a ZK proof `proof` given the public input `input` against the verification key
+    /// `identifier`.
     // IMPORTANT: this must match the extension ID in `extension_ids.rs`! However, because constants
     // are not inlined before macro processing, we can't use an identifier from another module here.
-    #[ink(extension = 42)]
+    #[ink(extension = 0)]
```

Comment:
Are CE's indexes global? i.e. if there was another CE then it'd need to know that 0 is already taken?

Reply:
if I understand correctly, they are now scoped

Reply:
if I understand correctly, they are now scoped